mdpow/equil.py

```diff
@@ -284,24 +296,47 @@ def topology(self, itp='drug.itp', prm=None, **kwargs):
         dirname = kwargs.pop('dirname', self.BASEDIR('top'))
         self.dirs.topology = realpath(dirname)
 
-        top_template = config.get_template(kwargs.pop('top_template', 'system.top'))
+        settings = {
+                    'OPLS-AA': ['oplsaa.ff/', 'oplsaa.ff/ions_opls.itp', 
+                                'oplsaa.ff/tip4p.itp'],
+                    'AMBER': ['amber99sb.ff/', 'amber99sb.ff/ions.itp', 
+                               'amber99sb.ff/tip3p.itp'],
+                    'CHARMM': ['charmm36-jul2017.ff/', 'charmm36-jul2017.ff/ions.itp', 
+                              'charmm36-jul2017.ff/tip3p.itp'],
+                    }
+        templates = {'water': 'system.top', 'octanol': 'system.top', 
```

Comment:
Can you add a test function that specifically tests that all the templates can be found?